### PR TITLE
Lab7

### PR DESCRIPTION
**Pull request by Maxkoz777**

### CODE REVIEW

**Comment by none-word** [comment]
Well done!

**Comment by afonyaa** [comment]
good job!

File: monitoring/promtail/promtail.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+server:
+    http_listen_port: 9080
+    grpc_listen_port: 0
```
Comment (AKurmazov) [inline monitoring/promtail/promtail.yml:3]: nit: You can remove this redundant statement. Also, you can specify the http_listen_address explicitly

**Comment by AKurmazov** [comment]
Approved 👍

File: monitoring/promtail/promtail.yml (hunk 1 above)
Comment (afonyaa) [reply monitoring/promtail/promtail.yml:3]: Good point, Alexander!

**Pull request closed without merging**